src/pages/index.tsx

```diff
@@ -18,11 +21,44 @@ const IndexPage: FunctionComponent = () => {
       />
       <main className="bg-gray-50 dark:bg-gray-900 sm:-my-5 -my-3 -mx-5 p-5">
         <div className="max-w-screen-xl mx-auto">
-          <Home />
+          <Home sections={sections} />
         </div>
       </main>
     </>
   )
 }
 
 export default IndexPage
+
+const digitalGardeningQuery = groq`
+*[_type == 'resource' && slug.current == "digital-gardening-for-developers"][0]{
+  title,
+  description,
+  'illustration': images[label == 'eggo'][0]{
+    url,
+    alt
+  },
+  'quote': content[title == 'quote'][0]{
+    description
+  },
+  resources[]{
+    title,
+    byline,
+    'name': content[title == 'name'][0].description,
+    'path': resources[]->[0].path,
+    'image': resources[]->[0].image
+  }
+}
+`
+
+export async function getStaticProps() {
+  const digitalGardeningFeature = await sanityClient.fetch(
+    digitalGardeningQuery,
+  )
+
+  return {
+    props: {
+      sections: [digitalGardeningFeature],
```

Comment:
making `sections` an array because we don't want to make the data coming in tied to the view